jazz_environment-event-handler/index.js

```diff
@@ -242,64 +241,37 @@ var processItem = function (eventPayload, configData, authToken) {
 }
 
 var processEventInitialCommit = function (environmentPayload, configData, authToken) {
-	var processStgEnv = new Promise((resolve, reject) => {
-		environmentPayload.logical_id = "stg";
-		environmentPayload.status = configData.CREATE_ENVIRONMENT_STATUS;
-
-		var svcPayloadStg = {
-			uri: configData.BASE_API_URL + configData.ENVIRONMENT_API_RESOURCE,
-			method: "POST",
-			headers: { Authorization: authToken },
-			json: environmentPayload,
-			rejectUnauthorized: false
-		};
-
-		logger.info("svcPayloadStg" + JSON.stringify(svcPayloadStg));
-		request(svcPayloadStg, function (error, response, body) {
-			if (response.statusCode === 200 && body && body.data) {
-				return resolve(null, body);
-			} else {
-				logger.error("Error creating stg environment in catalog: " + JSON.stringify(response));
-				return reject({
-					"error": "Error creating stg environment for " + environmentPayload.domain + "_" + environmentPayload.service + " in  catalog",
-					"details": response.body.message
-				});
-			}
-		});
-
-	});
-
-
-	var processProdEnv = new Promise((resolve, reject) => {
-		environmentPayload.logical_id = "prod";
-		environmentPayload.status = configData.CREATE_ENVIRONMENT_STATUS;
-
-		var svcPayloadProd = {
-			uri: configData.BASE_API_URL + configData.ENVIRONMENT_API_RESOURCE,
-			method: "POST",
-			headers: { Authorization: authToken },
-			json: environmentPayload,
-			rejectUnauthorized: false
-		};
-
-		logger.info("svcPayloadProd" + JSON.stringify(svcPayloadProd));
-		request(svcPayloadProd, function (error, response, body) {
-			if (response.statusCode === 200 && body && body.data) {
-				return resolve(null, body);
-			} else {
-				logger.error("Error creating prod environment in catalog: " + JSON.stringify(response));
-				return reject({
-					"error": "Error creating prod environment for " + environmentPayload.domain + "_" + environmentPayload.service + " in catalog",
-					"details": response.body.message
-				});
-			}
+	function processEnv(env) {
+		return new Promise((resolve, reject) => {
+			environmentPayload.logical_id = env;
+			environmentPayload.status = configData.CREATE_ENVIRONMENT_STATUS;
+
+			var svcPayloadProd = {
```

Comment:
nitpick: rename variable to `svcPayload` given function is no longer env specific